docker-compose.yml

```diff
@@ -104,7 +104,7 @@ services:
 
   # PEcAn executor, executes jobs launched from web
   executor:
-    image: pecan/executor:${PECAN:-latest}
+    image: pecan/executor:${PECAN:-develop}
```

Comment:
This is a bit misleading, because our `develop` container version is 3 months behind the 1.6.0 release. However, I put it here anticipating that we should start building containers more frequently in the future, so that `pecan/executor:develop` actually tracks the GitHub source code develop branch.

However, it's no problem to change it to `RC1.6.0` if that's preferred.

Reply:
my thinking is that we want to use latest so people can use `docker-compose up` to bring up the latest official version of PEcAn. I would like to start pushing automatically both latest and develop to docker hub. If people would like to use develop, they should be able to use `PECAN=develop docker-compose up` to bring up the latest develop stack.

No matter what we need to be consistent and make sure we use either latest or develop for all images (except for the externally managed images).

Reply:
I agree that we should use `latest` to be consistent...but I'm not going to make this change until we have a `pecan/executor:latest` container up on Docker Hub. Currently, the only options are `RC1.6.0` and `develop`, hence this change.